.travis.yml

```diff
@@ -27,6 +28,15 @@ opam: &OPAM
       sudo chown -R coq:coq /home/coq/${CONTRIB_NAME}
       opam install ${CONTRIB_NAME} -v -y -j ${NJOBS}
       "
+  - echo -e "${ANSI_YELLOW}Testing ${CONTRIB_NAME}...${ANSI_RESET}" && echo -en 'travis_fold:start:script\\r'
```

Comment:
This line should be removed, as it isn't really useful, and is duplicated (Travis only accepts one occurrence of `travis_fold:start:UNIQUE_ID`)

```suggestion
```